tutorials/plot_object_raw.py

```diff
@@ -32,7 +32,7 @@
 # Give the sample rate
 print('sample rate:', raw.info['sfreq'], 'Hz')
 # Give the size of the data matrix
-print('%s channels x %s samples' % (len(raw), len(raw.times)))
+print('%s channels x %s samples' % (raw.info['nchan'], len(raw.times)))
```

Comment:
maybe to keep it similar for both channels and samples `len(raw.ch_names)`?

Reply:
I'm feeling lazy :) Unless there is a strong reason, I'd keep it this way. I think this is the recommended way.

Reply:
ok :)